powerstore/resource_snapshotrule.go

```diff
@@ -71,8 +71,9 @@ func (r resourceSnapshotRuleType) GetSchema(_ context.Context) (tfsdk.Schema, di
 				Computed:            true,
 				Description:         "The time zone identifier for applying the time zone to the time_of_day for a snapshot rule.",
 				MarkdownDescription: "The time zone identifier for applying the time zone to the time_of_day for a snapshot rule.",
-				PlanModifiers: tfsdk.AttributePlanModifiers{
-					DefaultAttribute(types.String{Value: string(gopowerstore.TimeZoneEnumUTC)}),
+				PlanModifiers:       tfsdk.AttributePlanModifiers{
+					// DefaultAttribute(types.String{Value: string(gopowerstore.TimeZoneEnumUTC)}),
```

Comment:
if it not required then please remove it.

Reply:
what if in future reader wonder what could have been achieved by not using default attribute for this particular param but for others

so left a comment there in next line explaining the reason

Reply:
Then please mark it as TODO, we can not leave comments like this

Reply:
comments should be there as many as we can right ?
its not a TODO, its there to tell why we not including it